src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -170,6 +172,33 @@ def sagemaker_train(
         elif not is_pipe:
             # Check if there is potential data redundancy between training and validation sets
             check_data_redundancy(train_path, val_path)
+
+    # Obtain information about training resources to determine which distributed setup to use, if needed.
+    num_hosts = len(sm_hosts)
+
+    '''
+    tree_method_selected = validated_train_config.get("tree_method")
+    num_gpus = int(os.environ[SM_NUM_GPUS])
+
+    
+    # Use Dask for multi-GPU distributed training.
+    if eligible_for_dask_run(tree_method_selected, num_gpus, is_pipe, file_type):
+        logging.info("Will run distributed GPU training with Dask.")
+        run_training_with_dask(validated_train_config,
+                               train_path,
+                               val_path,
+                               model_dir,
+                               sm_hosts,
+                               sm_current_host,
+                               num_gpus)
+        return
+    '''
```

Comment:
At minimum (when uncommented), this should probably be moved out of this method.